app/main.py

```diff
@@ -1,13 +1,37 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"}
```

Comment:
fix the breaking line here, u can move closing quote to the next line or move "line" to the open quote